src/cow-react/modules/limitOrders/containers/LimitOrdersWidget/index.tsx

```diff
@@ -34,12 +36,13 @@ export function LimitOrdersWidget() {
   const onCurrencySelection = useOnCurrencySelection()
   const limitOrdersNavigate = useLimitOrdersNavigate()
   const limitOrdersQuote = useAtomValue(limitOrdersQuoteAtom)
+  const settingsState = useAtomValue(limitOrdersSettingsAtom)
 
   const currenciesLoadingInProgress = false
   const allowsOffchainSigning = false
   const isTradePriceUpdating = false
   const showSetMax = true
-  const showRecipientControls = true
+  const showRecipientControls = settingsState.showRecipient
```

Comment:
If all you need form `settingsState` is `showRecipient` I suggest to de-structure only that property from the atom and avoid the intermediary variable